UI/js/common.js

```diff
@@ -0,0 +1,25 @@
+/*
+UI errors for user display
```

Comment:
I tihnk that Sergey meant is that since you did
/**
 *
 */
below, we should use that as a convention for comments